03_homework/homework.sh

```diff
@@ -7,10 +7,14 @@
 # On your terminal, input all the commands you have used to create the following:
 
 # 1. How would you create 5 directories? Feel free to use any name for your directories.
+mkdir dir1 dir2 dir3 dir4 dir5
 
 # 2. How would you verify the creation of all 5 directories?
+ls
 
 # 3. In each directory, how would you create 5 .txt files and write "I love data" into each within the directories?
+touch file.txt
+cp file.txt dir1/
```

Comment:
You are only copying file.txt to dir1 and no specification of adding "I love data" into the file